dpimport_mriqc.sh

```diff
@@ -34,5 +34,5 @@ cd ${NDA_ROOT}
 import.py -c /data/predict/dpimport/examples/$CONFIG MRI_ROOT/derivatives/quick_qc/combined-*-mriqc-day1to*.csv
```

Comment:
Please enclose `MRI_ROOT/derivatives/quick_qc/combined-*-mriqc-day1to*.csv` in `" "` as well.